vignettes/datatable-intro.Rmd

```diff
@@ -250,6 +250,50 @@ The function `length()` requires an input argument. We just need to compute the
 
 This type of operation occurs quite frequently, especially while grouping (as we will see in the next section), to the point where `data.table` provides a *special symbol* `.N` for it.
 
+### g) Handle non-existing elements in `i`
+
+#### -- What happens when querying for non-existing elements?
+
+When querying a `data.table` for elements that do not exist, the behavior differs based on the method used.
+
+```r
+dt <- data.table(x = letters[1:3], y = LETTERS[1:3])
+setkeyv(dt, "x")
+```
+
+* **Key-based subsetting: `dt["d"]`**
+  
+  This performs a right join on the key column `x`, resulting in a row with `d` and `NA` for columns not found.
+
+  ```r
+  dt["d"]
+  # Returns:
+  #    x    y
+  # 1: d <NA>
+  ```
+
+* **Logical subsetting: `dt[x == "d"]`**
+  
+  This performs a plain subset operation that does not find any matching rows and thus returns an empty data.table.
```

Comment:
`data.table`

Reply:
'plain' doesn't add anything here (and if you still need to use an additional term then something like 'standard' fits more imo)

Reply:
done 👍